dhis-2/dhis-services/dhis-service-administration/src/main/resources/data-integrity-checks.yaml

```diff
@@ -42,6 +42,7 @@ checks:
   - programs/programs_custom_data_entry_form_empty.yaml
   - programs/tracker_geometry_invalid_srid.yaml
   - program_indicators/program_indicators_without_expression.yaml
+  - tracker/single_events_null_occurred_date.yaml
```

Comment:
I think the convention here is the first part determines the type of metadata. Since tracker is not a metadata type, we could change it to "programs/..." for example?

Reply:
Done